### PR TITLE
Make Advanced Parameters > Database > Backup form multistore compatible

### LINKED ISSUE #19310: Multistore - Advanced Parameters > Database > Backup - Add specific settings dropdown
As a merchant, I want to be able to see if a parameter has a different value for another store when I am in "all shops" context or in a group context, so I can know to which stores the change I am making will be applied

[Specs specific settings dropdown](https://github.com/PrestaShop/prestashop-specs/blob/9783996765a46d3d7197e4f7cd7098ffb7e37ff2/content/1.7/back-office/multistoregeneralspecs.md#specific-settings-drop-down-only-from-178)

### PR DESCRIPTION
**Pull request by ecomiz**
| Questions         | Answers
| ----------------- | -------------------------------------------------------
| Branch?           | develop
| Description?      | This PR add checkboxes and specifics settings drop-downs list to BO > Advanced Parameters > Database > Backup page to make it multistore compatible
| Type?             |  new feature
| Category?         | BO 
| BC breaks?        | no
| Deprecations?     | no
| Fixed ticket?     | Fixes #19310  and #19357.
| How to test?      | See issue
| Possible impacts? | BO > Advanced Parameters > Database > Backup

---
This change is [Reviewable](https://reviewable.io/reviews/prestashop/prestashop/26480)

### CODE REVIEW

File: admin-dev/themes/new-theme/js/pages/backup/index.ts [hunk 1]
```diff
@@ -42,3 +42,11 @@ $(() => {
   backupGrid.addExtension(new SubmitRowActionExtension());
   backupGrid.addExtension(new FiltersSubmitButtonEnablerExtension());
 });
+
+$(() => {
```
Comment (Progi1984) [inline admin-dev/themes/new-theme/js/pages/backup/index.ts]:
```suggestion

```

File: src/PrestaShopBundle/Form/Admin/Configure/AdvancedParameters/Backup/BackupOptionsType.php [hunk 2]
```diff
@@ -31,7 +31,7 @@
 use PrestaShopBundle\Form\Admin\Type\TranslatorAwareType;
 use Symfony\Component\Form\FormBuilderInterface;
 use Symfony\Component\Translation\TranslatorInterface;
-
+use PrestaShopBundle\Form\Admin\Type\MultistoreConfigurationType;
```
Comment (Progi1984) [inline src/PrestaShopBundle/Form/Admin/Configure/AdvancedParameters/Backup/BackupOptionsType.php:34]:
```suggestion
use PrestaShopBundle\Form\Admin\Type\MultistoreConfigurationType;

```

**Review by Progi1984 — COMMENTED** [review]
@ecomiz Thank you for your contribution. I have brought two small comments.

**Review by matthieu-rolland — COMMENTED** [review]
thank you for your contribution !

you need to rebase your PR, there are conflicts on two files

**Review by Progi1984 — DISMISSED** [review]

**Comment by Progi1984** [comment]
@ecomiz Sorry. I have approved so fast. Could you rebase your PR and remove your merge commit ?

**Comment by ecomiz** [comment]
> @ecomiz Sorry. I have approved so fast. Could you rebase your PR and remove your merge commit ?

Hello, I need help, how to remove the merge commit and rebase the PR ? Thank you

**Comment by Progi1984** [comment]
@ecomiz I think your rebase is wrong. Could you check it ?

**Comment by matks** [comment]
Hi @ecomiz,

Since we had no news from you for almost 2 months, I will follow [our guidelines](https://devdocs.prestashop.com/1.7/project/maintainers-guide/reviewing-pull-requests/#stale-pull-requests) and I'll close this pull request as it seems stale. Naturally if you want to continue this PR, you can reopen 😉  .

Thanks!

**Pull request closed without merging**